src/docs/README.md

```diff
@@ -137,31 +156,53 @@ Any file named `README.md` (case sensitive!) will not be used to generate html p
 
 ## Structure and Format
 
-- Max. 3 headline levels (## & ###)
+Try to avoid deep structures by using no more than two headline levels, i.e.
+
+<pre><code>```
+## Headline level 2  
+### Headline level 3
+```</code></pre>
 
+Level 1 is automatically generated from the article's `title: `. 
+ 
 ## Images
 
 - simple markdown logic (not HTML): `![name_of_image](path_to_image.png)`
 
 ## File Naming
 
-Preferred: 
-- [snake_case](https://en.wikipedia.org/wiki/Snake_case) (lower_case_with_underscores) 
+Avoid
```

Comment:
Rephrase this to `File Name Requirements` and follow requirement level naming: https://www.ietf.org/rfc/rfc2119.txt